gapic-generator-java/src/main/java/com/google/api/generator/gapic/model/GapicContext.java

```diff
@@ -59,6 +60,8 @@ public GapicMetadata gapicMetadata() {
   @Nullable
   public abstract com.google.api.Service serviceYamlProto();
 
+  public abstract boolean isEmpty();
```

Comment:
Similar to above in PackageInfoDefinition. Would it be possible to expose a method like

```
public boolean containsServices() {
    return !services().isEmpty();
}
```

instead of adding a new field to GapicContext? If not, I think this is fine.

Reply:
Done